operate/importer/src/main/java/io/camunda/operate/zeebeimport/ImportPositionHolder.java

```diff
@@ -43,7 +43,8 @@ public class ImportPositionHolder {
       new HashMap<>();
   private final Map<String, ImportPositionEntity> inflightImportPositions = new HashMap<>();
   private final Map<String, ImportPositionEntity> inflightPostImportPositions = new HashMap<>();
-  private final Map<String, Gauge> importPositionCompletedGauges = new HashMap<>();
+  private final ConcurrentHashMap<String, Boolean> mostRecentProcessedImportPositions =
```

Comment:
❓ Wondering about this, as this would mean that the other maps are also not safe?!

Reply:
Was this now the issue or the other fix? 😃

Reply:
This is an artifact from a prior solution I tried doesn't need to be this way I'll make it a normal map